src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ResolverHandler.java

```diff
@@ -167,26 +171,26 @@ public static void removeResolver(final IStandardRequestManager manager, final I
      *
      * @throws IllegalArgumentException is thrown when the given token is not registered to any IRequestResolver
      */
-    public static IRequestResolver getResolver(final IStandardRequestManager manager, final IToken token) throws IllegalArgumentException
+    public static IRequestResolver<? extends IRequestable> getResolver(final IStandardRequestManager manager, final IToken<?> token)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1452)